### PR TITLE
Add Storage Buckets article

### PR DESCRIPTION
**Pull request by tomayac**
Fixes https://github.com/GoogleChrome/developer.chrome.com/issues/4135

Changes proposed in this pull request:

- Add the article.

Please review in the [Google doc](https://docs.google.com/document/d/1qpnZZ_sYWHjVFkO8G1MRU4pduxvTEf55sEZSrfanJiU/edit?resourcekey=0-v_5H-v39qp-T6g57S2auGA#). You can also see the [staging preview](https://deploy-preview-4139--developer-chrome-com.netlify.app/blog/storage-buckets/).

### CODE REVIEW

File: site/en/blog/storage-buckets/index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Not all storage is created equal: introducing Storage Buckets'
+subtitle: >
+  When the browser is under heavy storage pressure, it can evict your least important storage
```
Comment (rachelandrew) [inline site/en/blog/storage-buckets/index.md:5]: this subtitle description is a bit odd, it's a problem statement. Maybe, "We're experimenting with an API for persistent storage, and we'd love your feedback."

**Review by rachelandrew — COMMENTED** [review]
Can we just make the subhead a bit more descriptive rather than the current problem statement, which looks a bit strange.

File: site/en/blog/storage-buckets/index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  buckets first, and keep the mission-critical ones around.
```
Comment (tomayac) [inline site/en/blog/storage-buckets/index.md:6]:
```suggestion
  We're experimenting with an API for making persistent storage eviction under heavy
  memory pressure more predictable, and we'd love your feedback.
```

File: site/en/blog/storage-buckets/index.md (hunk 1 above)
Comment (tomayac) [reply site/en/blog/storage-buckets/index.md:5]: What do you think of this suggestion? https://github.com/GoogleChrome/developer.chrome.com/pull/4139/files#r1013974736

File: site/en/blog/storage-buckets/index.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Not all storage is created equal: introducing Storage Buckets'
+subtitle: >
+  We're experimenting with an API for making persistent storage eviction under heavy
+  memory pressure more predictable, and we'd love your feedback.
+description: >
+  When the browser is under heavy storage pressure, it can evict your least important storage
+  buckets first, and keep the mission-critical ones around.
```
Comment (rachelandrew) [inline site/en/blog/storage-buckets/index.md:9]:
```suggestion
  We're experimenting with an API for making persistent storage eviction under heavy
  memory pressure more predictable, and we'd love your feedback.
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**